Fix svl script

nseidle opened the pull request:
This adds back serial exception printing.

It also adds a few helper texts:

![image](https://user-images.githubusercontent.com/117102/75587213-365b3380-5a33-11ea-857a-655809ad84e3.png)

Below - when COM is valid but the target fails to respond to the 'enter bootload' command.
![image](https://user-images.githubusercontent.com/117102/75587266-54c12f00-5a33-11ea-8ccd-3902b150ad67.png)

Below - verbose output when COM is valid but target fails to respond:
![image](https://user-images.githubusercontent.com/117102/75587343-77ebde80-5a33-11ea-8278-e5f700f52a9d.png)

oclyke commented:
@nseidle Thanks!

* Seems to be a typo in ```Bin file blank.bin does not **exits**```
* When the target fails to enter bootload mode... this could also mean that the SVL is not running on the board. Should we add a message like ```Verify right COM port is selected **and that your board has the SVL bootloader**``` (or is that TMI for the user?

nseidle commented:
Good typo eyes. Fixed. And I'm not opposed to it.

Fixes in place. I added a version number to verbose output. Exe has been compiled.

![image](https://user-images.githubusercontent.com/117102/75829880-96701380-5d6c-11ea-8b18-dc2396ef6f72.png)
